frappe/core/doctype/rq_worker/rq_worker.json

```diff
@@ -76,13 +76,13 @@
   {
    "fieldname": "queue",
    "fieldtype": "Data",
-   "label": "Queue"
+   "label": "Queue(s)"
   },
   {
    "fieldname": "queue_type",
    "fieldtype": "Select",
    "in_list_view": 1,
-   "label": "Queue Type",
+   "label": "Queue Type(s)",
    "options": "default\nlong\nshort"
```

Comment:
remove the options? and make it a data field?

doesn't really matter much as it's virtual doctype anyway :P

Reply:
Didn't do it cause it looks nicer on list view with pill :)